Makefile

```
@@ -183,7 +183,7 @@ JV ?= $(LOCALBIN)/jv
 
 ## Tool Versions
 KUSTOMIZE_VERSION ?= v5.2.1
-CONTROLLER_TOOLS_VERSION ?= v0.13.0
+CONTROLLER_TOOLS_VERSION ?= v0.14.0
```

It seems we missed this when upgrading to go1.22 and kubebuilder. Maybe it would be better to upgrade to the latest v0.15.0 that uses the same client-go as we do. 
```suggestion
CONTROLLER_TOOLS_VERSION ?= v0.15.0
```

---

changed to `v0.15.0` now